core/quill.js

```diff
@@ -227,6 +234,11 @@ class Quill {
     this.setContents(delta, source);
   }
 
+  toggleEmpty(empty = false) {
```

Comment:
Can `quill.root.classList.toggle` be used instead of adding a new `toggleEmpty` function? They seem to accomplish the same things.